indico/modules/events/registration/templates/display/participant_list.html

```diff
@@ -11,6 +11,14 @@
     {% endif %}
 {% endblock %}
 
+{% block description %}
+    {%- if tables|length > 1 -%}
+        {% trans -%}
+            The lists of participants grouped by the registration form they used to registered to the event.
```

Comment:
So by adding "The **public** lists of participants..." should fix it :smile: 
There is nothing wrong with adding an 1-line description in the beginning of the page in order to remove any doubts of the users.